packages/utils/typescript/lib/generators/common/models/attributes.js

```diff
@@ -114,21 +114,21 @@ const getAttributeModifiers = (attribute) => {
       throw new Error('typeof min/max values mismatch');
     }
 
-    const typeofMinMax = (max && typeofMax) ?? (min && typeofMin);
     let typeKeyword;
```

Comment:
Do we want to explicitly define this as undefined and/or comment it so it's clear what's happening? I was looking for the `if(typeofMin === 'undefined')` down below and then had to look up to see why that case wasn't there